docs/contributing/community.md

```diff
@@ -8,17 +8,39 @@ Follow and interact with us on Twitter: [@ThanosMetrics](https://twitter.com/Tha
 
 ### Thanos Community Office Hours
 
-Thanos Community Office Hour (previously Contributor Office Hour) is a public weekly meeting focused on Thanos community, development and contributions. It’s meant for users, developers and maintainers to meet and get unblocked, pair review, and discuss development aspects of Thanos and related projects. **Everyone is welcome to join & add items to the agenda**. [CoC applies](../../CODE_OF_CONDUCT.md).
+Thanos Community Office Hour (previously Contributor Office Hour) is a public weekly meeting focused on Thanos community, development and contributions. It’s meant for users, developers and maintainers to meet and get unblocked, pair review, and discuss development aspects of Thanos and related projects. 
 
-* When: Alternating weekly on Thursday:
-  * One week EU friendly time: 11:00-12:00 UTC
-  * One week US friendly time: 17:00-18:00 UTC
-* Calendar Event: https://www.cncf.io/calendar/
-* Agenda: [Google Doc](https://docs.google.com/document/d/137XnxfOT2p1NcNUq6NWZjwmtlSdA6Wyti86Pd6cyQhs)
-* Link to join zoom webinar: [https://zoom.us/j/96037860612](https://zoom.us/j/96037860612)
+**Everyone is welcome to join & add items to the agenda**.
 
 > NOTE: If no agenda items will be added 1h before the meeting, the meeting will be cancelled.
 
+**Meeting Time:** Every-second Thursday 15:00 CET
```

Comment:
Hour vote still pending (https://cloud-native.slack.com/archives/CL25937SP/p1659615345961049)

Reply:
Looks like 3PM still has the most votes, although there is some dissent in the thread

Reply:
What happens when the timezone changes from CET to CEST? We have to remember to come here and update this? 

What about writing time always in UTC to make it friendly for people from different timezones to understand at what time the meeting happens? It's easy to convert UTC to my "my own" timezone and it's much more difficult to convert "other random timezone" to "my own".

Even if it's decided that the meeting will happen in different times in European summer or winter, writing it down in UTC is still the most friendly way.

Reply:
Yea, I am always confused with UTC, plus it changes to maintainer team which is mostly CET (: - happy to change though.